vm_manager/vm_manager_cluster.py

```diff
@@ -671,6 +741,33 @@ def clone(vm_options_with_nones):
         vm_options["preferred_host"]
     ):
         raise ValueError(f"{vm_options['preferred_host']} is not valid hypervisor")
+    for pacemaker_arg in (
+        "pacemaker_meta",
+        "pacemaker_param",
+        "pacemaker_utilization",
+    ):
+        clear_arg = vm_options.get(f"clear_{pacemaker_arg}", False)
+        if not clear_arg:
+            pacemaker_new_arg = vm_options.get(pacemaker_arg, {})
```

Comment:
Won't work for `clear_pacemaker_param` because it has an s in cretae/clone commands, `pacemaker_params`